client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -154,63 +211,51 @@ class JetpackChecklist extends PureComponent {
 							target="_blank"
 						/>
 					) }
-					{ map( taskStatuses, ( status, taskId ) => {
-						const task = JETPACK_SECURITY_CHECKLIST_TASKS[ taskId ];
-
-						if ( ! task ) {
-							// UI does not support this task.
-							return;
-						}
-
-						return (
-							<Task
-								completed={ get( status, 'completed', false ) }
-								completedButtonText={ task.completedButtonText }
-								completedTitle={ task.completedTitle }
-								description={ task.description }
-								duration={ task.duration }
-								href={ task.getUrl( siteSlug ) }
-								onClick={ this.handleTaskStart( {
-									taskId,
-									tourId: get( task, 'tourId', null ),
-								} ) }
-								title={ task.title }
-								key={ taskId }
-							/>
-						);
-					} ) }
+					{ this.renderTaskSet( JETPACK_SECURITY_CHECKLIST_TASKS ) }
 					{ /* For Checklist completion calculation to work correctly, children shold be a flat list of tasks */ }
 					{ isEnabled( 'jetpack/checklist/performance' ) &&
 						this.renderSectionTitle( translate( 'Performance Tools' ) ) }
-					{ isEnabled( 'jetpack/checklist/performance' ) && (
-						<Task title="Static task for demonstration purposes." completed />
-					) }
 					{ isEnabled( 'jetpack/checklist/performance' ) &&
-						map( taskStatuses, ( status, taskId ) => {
-							const task = JETPACK_PERFORMANCE_CHECKLIST_TASKS[ taskId ];
-
-							if ( ! task ) {
-								// UI does not support this task.
-								return;
+						this.renderTaskSet( JETPACK_PERFORMANCE_CHECKLIST_TASKS ) }
+					{ isEnabled( 'jetpack/checklist/performance' ) && ( isPremium || isProfessional ) && (
+						<Task
+							title={ translate( 'Video Hosting' ) }
+							description={ translate(
+								'Enable fast, high-definition, ad-free video hosting through our global CDN network.'
+							) }
+							completedButtonText={ translate( 'Upload videos' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Upload Videos', { context: 'Media upload plan needed'} )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).